### PR TITLE
Add code coverage options for gathering CC results in validation

### PR DESCRIPTION
**Pull request by carlbrochu**
Add settings to include our binaries and filter other binaries to enable better code coverage results.
Add a few tests around livy job 
Update sln file for existing project+missing project

### CODE REVIEW

File: Services/DataX.Config/DataX.Config.LivyClient.Test/JobOperationTest.cs [hunk 1]
```diff
@@ -36,6 +36,52 @@ public async Task TestStartJob()
...
+        }
+
+        [TestMethod]
+        public async Task TestStopJob()
+        {
+            var appInfoJson = "\"appInfo\":{\"driverLogUrl\":\"http://123.xx.internal.cloudapp.net:3/node/containerlogs/container_123/livy\",\"sparkUiUrl\":\"https://site.azurehdinsight.net/yarnui/hn/proxy/application_123_0101/\"}";
+            var logs1 = ",\"log\":[\"\\t user: livy\",\"19/01/30 04:02:31 INFO ShutdownHookManager: Shutdown hook called\",\"\\nYARN Diagnostics: \"]}";
+            var logs2 = ",\"log\":[\"\\t user: livy\",\"19/01/30 04:02:31 INFO ShutdownHookManager: Shutdown hook called\",\"19/01/30 04:02:31 INFO ShutdownHookManager: Deleting directory /tmp/spark-123\",\"19/01/30 04:02:31 INFO ShutdownHookManager: Deleting directory /tmp/spark-123\",\"19/01/30 04:02:31 INFO MetricsSystemImpl: Stopping azure-file-system metrics system...\",\"19/01/30 04:02:31 INFO MetricsSinkAdapter: azurefs2 thread interrupted.\",\"19/01/30 04:02:31 INFO MetricsSystemImpl: azure-file-system metrics system stopped.\",\"19/01/30 04:02:31 INFO MetricsSystemImpl: azure-file-system metrics system shutdown complete.\",\"\\nstderr: \",\"\\nYARN Diagnostics: \"]}";
```
Comment (kjcho-msft) [inline Services/DataX.Config/DataX.Config.LivyClient.Test/JobOperationTest.cs]: looks like this is not used.

File: Services/DataX.Config/DataX.Config.LivyClient.Test/JobOperationTest.cs [hunk 2]
```diff
@@ -36,6 +36,52 @@ public async Task TestStartJob()
             Assert.AreEqual(expected: JobState.Running, actual: refreshJobInfo.JobState);
         }
 
+        [TestMethod]
+        public async Task TestGetsJobs()
```
Comment (kjcho-msft) [inline Services/DataX.Config/DataX.Config.LivyClient.Test/JobOperationTest.cs]: should this be just TestGetJobs?

Comment (carlbrochu) [reply Services/DataX.Config/DataX.Config.LivyClient.Test/JobOperationTest.cs]: yes

File: Services/DataX.Config/DataX.Config.LivyClient.Test/JobOperationTest.cs (hunk 1 above)
Comment (carlbrochu) [reply Services/DataX.Config/DataX.Config.LivyClient.Test/JobOperationTest.cs]: removed

**Pull request merged**